src/engine/platform/fmshared_OPN.h

```diff
@@ -20,17 +20,10 @@
 #ifndef _FMSHARED_OPN_H
 #define _FMSHARED_OPN_H
 
-#define ADDR_MULT_DT 0x30
-#define ADDR_TL 0x40
-#define ADDR_RS_AR 0x50
-#define ADDR_AM_DR 0x60
-#define ADDR_DT2_D2R 0x70
-#define ADDR_SL_RR 0x80
-#define ADDR_SSG 0x90
-#define ADDR_FREQ 0xa0
-#define ADDR_FREQH 0xa4
-#define ADDR_FB_ALG 0xb0
-#define ADDR_LRAF 0xb4
+#include "fmsharedbase.h"
+
+#define CHIP_FREQBASE fmFreqBase
```

Comment:
Definr being in a header may cause troubles... Try inserting this on the .cpp files instead.